### PR TITLE
Add ICP

### PR DESCRIPTION
**Pull request by nathanosdev**
https://github.com/dfinity/awesome-icp#readme

This list relates to the Internet Computer, the world’s first blockchain that runs at web speed and can increase its capacity without bound. Like the Internet (which is composed of many machines adhering to TCP/IP protocol) and blockchain protocols (such as Bitcoin and Ethereum). It has a thriving community of developers with a growing toolset and adding it to this list would help support these developers in discovering the tools that they need to develop on the IC.

Reviewed PRs:
- https://github.com/sindresorhus/awesome/pull/2389#issuecomment-1253917789
- https://github.com/sindresorhus/awesome/pull/2386#issuecomment-1253953714
- https://github.com/sindresorhus/awesome/pull/2391#issuecomment-1254864005

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

### CODE REVIEW

**Comment by n0kovo** [comment]
Sorry for not being constructive here, but I just have to mention that for a second I thought this was a list of Insane Clown Posse resources 😅

**Review by Kuensewe — APPROVED** [review]

**Comment by cosydney** [comment]
One of the pre-requesites of making a PR is not to add 'Awesome' in your PR title

**Comment by nathanosdev** [comment]
Thanks @cosydney, it's updated now!

**Comment by Jogai** [comment]
Read the pr template again

**Comment by nathanosdev** [comment]
@Jogai I've read the template multiple times, believe me. I think comments suggesting that someone should read the template again are only useful when it's blatantly obvious that they haven't bothered to read the guidelines at all. I think it's clear that I actually have read the guidelines, but if something has fallen through the cracks then I'd appreciate it if you could point it out.

**Review by gpotter2 — COMMENTED** [review]
Cool list, it's pretty well built.

Small tip: `<a align="center">` doesn't work, you should replace

```
<a align="center" href="https://internetcomputer.org/">
  <img src="assets/dfinity-logo.png">
</a>
```
with
```
<p align="center">
  <a href="https://internetcomputer.org/"><img src="assets/dfinity-logo.png"></a>
</p>
```

File: readme.md [hunk 1]
```diff
@@ -791,6 +791,7 @@
 - [Golem](https://github.com/golemfactory/awesome-golem#readme) - Open source peer-to-peer marketplace for computing resources.
 - [Stacks](https://github.com/friedger/awesome-stacks-chain#readme) - A smart contract platform secured by Bitcoin.
 - [Algorand](https://github.com/aorumbayev/awesome-algorand#readme) - An open-source, proof of stake blockchain and smart contract computing platform.
+- [Internet Computer](https://github.com/dfinity/awesome-icp#readme) - A public blockchain that hosts smart contracts that run at web speed.
```
Comment (sindresorhus) [inline readme.md:794]: The repo title should be `awesome-internet-computer`. Same with the pull request title.

**Comment by sindresorhus** [comment]
Some entries are missing a description.

**Comment by sindresorhus** [comment]
`ViteJS` => `Vite`

**Comment by arkangelg69** [comment]
I pulled all my asset's. Still lost over 140k usd

**Comment by arkangelg69** [comment]
Im  uninstall it all on my end

**Comment by arkangelg69** [comment]
Naa  i had money here and mempool lighting conductor. But moved to a wallet that doesn't support decentralized bitcoin . Lost it all

**Comment by arkangelg69** [comment]
Goodbye fellas

File: readme.md (hunk 1 above)
Comment (nathanosdev) [reply readme.md]: I've changed the title to ICP. If I understand correctly, the issue is that the title here and the repo need to match.

**Comment by nathanosdev** [comment]
> Some entries are missing a description.
Fixed.

> `ViteJS` => `Vite`
Fixed.

Thank you for the feedback @sindresorhus

**Pull request closed without merging**